docs/book/getting-started/database-and-models.md

```diff
@@ -361,63 +398,53 @@ class AlbumController extends AbstractActionController
 <!-- markdownlint-enable MD033 -->
 
 Our controller now depends on `AlbumTable`, so we will need to create a factory
-for the controller.  Similar to how we created factories for the model, we'll
-create it in our `Module` class, only this time, under a new method,
-`Album\Module::getControllerConfig()`:
+for the controller that will inject the `AlbumTable`.
 
-<!-- markdownlint-disable MD033 -->
-<pre class="language-php" data-line="12-24"><code>
-namespace Album;
+Let's create the `AlbumControllerFactory.php` in `module/Album/src/Controller`:
 
-use Laminas\Db\Adapter\AdapterInterface;
-use Laminas\Db\ResultSet\ResultSet;
-use Laminas\Db\TableGateway\TableGateway;
-use Laminas\ModuleManager\Feature\ConfigProviderInterface;
+````php
+namespace Album\Controller;
+
+use Album\Model\AlbumTable;
+use Laminas\ServiceManager\Factory\FactoryInterface;
+use Psr\Container\ContainerInterface;
 
-class Module implements ConfigProviderInterface
+class AlbumControllerFactory implements FactoryInterface
```

Comment:
I would skip this factory because the main goal is to get started quickly and easily. The creation of boilerplate code must be avoided.
We must bear in mind that the getting started tutorial also has an advertising function.

Reply:
Ah sorry, the `ReflectionBasedAbstractFactory` from above.

Reply:
The factory for the class `AlbumTable` is correct.

Reply:
Done